pkg/v2/resource/chart/resource.go

```diff
@@ -101,6 +105,36 @@ func (r *Resource) Name() string {
 	return Name
 }
 
+func (r *Resource) getGuestHelmClient(ctx context.Context, obj interface{}) (helmclient.Interface, error) {
+	clusterGuestConfig, err := r.toClusterGuestConfigFunc(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", "looking for chart-operator chart in the guest cluster")
```

Comment:
I think this log message is in wrong place?

Reply:
Yep, removed.